common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -82,12 +82,34 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
 
   }
 
+  def cleanAmpInteractives(document: Document) = {
+
+    document.getElementsByClass("element-interactive").filter { element: Element =>
+      element.getElementsByTag("a").length !=0
+    }.foreach { interactive: Element =>
+      val link = interactive.getElementsByTag("a")
+      val linkToInteractive = link.first().attr("href")
+      val iframe = document.createElement("amp-iframe")
+      // AMP uses the height and width ratio, so exact values don't matter
+      // All interactives should resize to the correct height once they load
+      iframe.attr("width", "5")
```

Comment:
So sorry, that's me not paying attention.

Thanks for the clarity :crystal_ball: